editor/store/selectors.js

```diff
@@ -1277,7 +1312,7 @@ export function isSavingReusableBlock( state, ref ) {
  * @return {Array} An array of all reusable blocks.
  */
 export function getReusableBlocks( state ) {
-	return Object.values( state.reusableBlocks.data );
+	return Object.keys( state.reusableBlocks.data ).map( ( ref ) => getReusableBlock( state, ref ) );
```

Comment:
Minor: Lodash's `_.map` works well on objects for this purpose (also trivially more performant by avoiding a second iteration):

```js
return map(
	state.reusableBlocks.data,
	( data, ref ) => getReusableBlock( state, ref )
);
```

Reply:
👌 changed in 7d87636ec2d95a9e678423fa66390ee2649af4ad.